src/fides/api/ops/schemas/user_permission.py

```diff
@@ -36,8 +35,12 @@ class UserPermissionsResponse(UserPermissionsCreate):
 
     id: str
     user_id: str
-    scopes: List[ScopeRegistryEnum]
-    total_scopes: List[ScopeRegistryEnum]
+    scopes: List[
+        ScopeRegistryEnum
+    ]  # This should be removed once the UI is not using.  Currently returning "total_scopes" here.
```

Comment:
I'm pretty sure the UI stopped using `scopes` [in this PR](https://github.com/ethyca/fides/pull/2747)